cpp/core/memory/ColumnarBatch.h

```diff
@@ -47,7 +48,8 @@ class ColumnarBatch {
   virtual std::shared_ptr<ArrowSchema> exportArrowSchema() = 0;
 
   virtual int64_t getExportNanos() const;
-  ;
+
+  virtual std::pair<char*, int> getFirstRowBytes() const;
```

Comment:
Would it be better to use `getRowBytes(int32_t rowId)`?

Then we can change the calling code to `getRowBytes(0)`

Reply:
@zhztheplayer Yes. Updated.

Reply:
Thanks!